### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

Can you please review my slider component?
The site is not responsive yet.
Link - https://naughty-joliot-2dd852.netlify.app/components/slider.html

Thanks

### CODE REVIEW

**Review by ark2002 — APPROVED** [review]
looks good to merge

**Review by akash-1318 — COMMENTED** [review]
No changes required looks good to merge.

File: Components/slider.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "/main.css">
+
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <title>Slider</title>
```
Comment (arjunbharti) [inline Components/slider.html:13]: You can added comments here. Like, what's the purpose of this link tag, etc. It enhances the code readability.

**Review by arjunbharti — APPROVED** [review]
The UI looks really nice. Just added a minor suggestion, have a look!

**Pull request merged**